.starters/default/.env.example

```diff
@@ -0,0 +1,3 @@
+# Create one with no scope selected on https://github.com/settings/tokens/new
+# This token is used for fetching the repository releases.
+GITHUB_TOKEN=
```

Comment:
Is it necessary? 🤔

Reply:
It was present in the previous Alpine starter, but I guess not.

Removing it!